design-proposals/configurable-features.md

```diff
@@ -0,0 +1,161 @@
+# Overview
+
+With the introduction
+of [KubeVirt Feature Lifecycle](https://github.com/kubevirt/community/blob/main/design-proposals/feature-lifecycle.md)
+policy, features reaching General Availability (GA) need to drop their use of feature gates. This applies also to
+configurable features that we may want to disable.
+
+## Motivation
```

Comment:
Is the idea of the proposal to outline a general way of handling kubevirt configurables, or is it specifically for VMI features?

Reply:
This proposal targets VM/VMI features respectively every configurable that needs to be taken into account when handling VMs/VMIs.